nncf/openvino/optimized_functions/functions.py

```diff
@@ -168,6 +169,39 @@ def quantize_dequantize_weight(
         return decompressed_weight
 
 
+def get_integer_quantization_error(
+    weight: Tensor,
+    reduction_axes: ReductionAxes,
+    config: WeightCompressionConfig,
+) -> float:
+    """
+    Calculates a quantity characterizing the difference between floating point weights and fake quantized
```

Comment:
Could you please add a clearer description because the code with building the openvino model is difficult to understand how "a quantity" is calculated. If you add the formula it will greatly improve the clarity.

Reply:
Added